updating guide: rename`--dumps-dir` to `--dump-dir` (#2371)
meilisearch/documentation: learn/update_and_migration/updating.mdx

  - The `private` and `public` keys have been deprecated and replaced by two default API keys with similar permissions: `Default Admin API Key` and `Default Search API Key`.
  - The `updates` API has been replaced with the `tasks` API.
- If you are **updating from v0.27 or below**, existing keys will have their `key` and `uid` fields regenerated.

> - If you are **updating from v0.30 or below on a cloud platform**, update the following files to rename `--dumps-dir` to `--dump-dir`:

Author: ```suggestion
- If you are **updating from v0.30 or below to v1.0 or above on a cloud platform**, update the following files to rename `--dumps-dir` to `--dump-dir`:
```

> - If you are **updating from v0.30 or below to v1.0 or above on a cloud platform**, update the following files to rename `--dumps-dir` to `--dump-dir`:

Reviewer: ```suggestion
- If you are **updating from v0.30 or below to v1.0 or above on a cloud platform**, replace `--dumps-dir` with `--dump-dir` in the following files:
```
